models/calendar.yml

```diff
@@ -130,6 +130,7 @@
         person:
           name: Alaina Kafkes
           twitter: alainakafkes
+          pronouns: # examples: she/her, they/them, he/him
```

Comment:
Looks good to me!  But I also know the context.

You could consider linking to some kind of "why are we doing this" somewhere on the site if you think the question is going to come up repeatedly.  This is a pretty good primer on Pronouns 101:
https://medium.com/trans-talk/trans-pronoun-faq-part-1-e96b5d10d425